tests/test_holiday_groups.py

```
@@ -129,3 +130,25 @@ def __init__(self, *args, **kwargs):
         test_holidays._add_visakha_bucha("Visakha Bucha")
         test_holidays._add_visakha_bucha("Visaka Bochea", KHMER_CALENDAR)
         self.assertEqual(0, len(test_holidays))
+
+
+class TestTibetanCalendarHolidays(TestCase):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)